src/Raven.Studio/typescript/commands/database/documents/getCollectionFieldsCommand.ts

```diff
@@ -0,0 +1,23 @@
+import commandBase = require("commands/commandBase");
+import database = require("models/resources/database");
+import endpoints = require("endpoints");
+
+class getCollectionFieldsCommand extends commandBase {
+
+    constructor(private database: database, private collectionName: string, private prefix: string) {
+        super();
+    }
+
+    execute(): JQueryPromise<object> {
```

Comment:
you can use dictionary<string> type here and in all usages.

Reply:
Changed to dictionary<string>. Thanks.